modules/detect_target/detect_target_ultralytics.py

```diff
@@ -108,4 +113,8 @@ def run(
         if self.__show_annotations:
             cv2.imshow("Annotated", image_annotated)  # type: ignore
 
+        end_time = time.time()
+        self.__logger.info(
+            f"{time.localtime()}: Target detection took {round(end_time - start_time, 3)} seconds"
```

Comment:
For time, it's fine to just use time.time() (logger automatically puts a timestamp with a proper format like hh:mm:ss), and there's no need to round it since we would rather have more precision. (these tasks usually take much less than 1 second, so more precision is required)